# Define the goal
uint32 command_id  # Specify which command
float32 p1  # parameter 1
float32 p2  # parameter 2
float32 p3  # parameter 3
float32 p4  # parameter 4
---
# Define the result
bool succeded
---
# Define a feedback message
float32 percent_complete

